tests/Util/CustomLayoutManager.php

```diff
@@ -0,0 +1,50 @@
+<?php
+declare(strict_types=1);
+
+namespace IntegerNet\GlobalCustomLayout\Test\Util;
```

Comment:
`tests/Util` is OK, but I recently switched to the convention `tests/src` as directory for fake classes, since it is source code, needed for tests (Suggested by Sebastian Bergmann). Feels more appropiate.